src/hooks/index.ts

```diff
@@ -13,6 +13,7 @@ export {
 // TODO: Have to properly remove these as we're no longer using them internally.
 export { useThemeProps, usePropsWithComponentTheme } from './useThemeProps';
 // TODO: instead export only this.
+export { useHasResponsiveProps } from '../hooks/useHasResponsiveProps';
```

Comment:
This is a deprecated hook.

Reply:
We are working on a new function that provides the same functionality, will release it shortly.